packages/openapi-generator/src/file-serializer/schema-file.spec.ts

```diff
@@ -139,14 +143,16 @@ describe('schemaFile', () => {
               required: true,
               schema: {
                 type: 'string'
-              }
+              },
+              typeProperties: {}
             },
             {
               name: 'string-property-no-description',
               required: true,
               schema: {
                 type: 'string'
-              }
+              },
+              typeProperties: {}
```

Comment:
also here add a test that the generated codeblock contains the new properties like @deprecated or @example and some other docs.